BranchMetrics/react-native-branch-deep-linking-attribution
# [SDK-2249] Expose the setDMAParamsForEEA method

## Reference
SDK-2249 -- Expose the setDMAParamsForEEA method

## Summary
Expose the setDMAParamsForEEA method in the JS layer and bump the native SDK versions.

## Motivation
So clients can utilize the method in React Native and stay compliant.

## Type Of Change


## Testing Instructions

Call the method like `Branch.setDMAParamsForEEA(true, true, true);` and observe the following request payloads for the respective DMA fields.









cc @BranchMetrics/saas-sdk-devs for visibility.
